### DISCUSSION TITLE
Does Spotube violate Spotify TOC?

### DISCUSSION THREAD

**Discussion by Robio** [KRTirtho/spotube#951] [Q&A] [answered]
Can't seem to find an answer to this anywhere. Can Spotify tell if I'm listening via Spottube? Can Spotify tell I'm bypassing ads? Can Spotify tell if I use Spotube to download? In short, is there anything about using this app that's going to hit me with the ban hammer?

**Comment by ildar** [comment]
I understand your concern but IMHO you should put IT to the subject, e.g.,
"can using Spotube lead me to ban?"

**Reply by Robio** [reply]
If you mean I should change the subject of this topic, I think the original subject is more direct, and it's the question I actually want answered: Does using Spotube violate Spotify's TOC?

**Comment by ildar** [comment]
IMHO that sounds more like a juristical question. And I guess there aren't
many jurists here reading.

**Reply by Robio** [reply]
I'm not looking for legal advice. I'm asking for anecdotal information, and maybe insight from people who know something about developing this kind of app — you know, the kind of people who have GitHub accounts where their software is publicly available for scrutiny. Three out of the 4 questions I asked could be answered by the developer in 10 seconds. The last is something anyone who knows the answers to the first three questions would be able to answer with an educated guess.

**Comment by fa-se** [comment, marked as answer]
In my understanding, Spotube only uses Spotify for its metadata and for your playlists etc.. The actual music is played from Youtube. It doesn't directly bypass ads from Spotify and it doesn't download music from Spotify.
While this answer doesn't fully answer your question, it might alleviate your concerns.

**Reply by Robio** [reply]
Thank you for actually answering the question!

I eventually figured this out for myself, and probably should have returned here to say so. Clever little app. Quite a bit on the buggy side, but it does it's job damn well for a jury-rigged solution, and looks good doing it! Far better UI than the Spotify app itself. Better UI than _most_ music players for Android, in fact.
